src/main/webapp/app/exercises/programming/hestia/git-diff-report/git-diff-file-panel.component.ts

```
@@ -2,57 +1,66 @@
-            .flatMap((entry) => {
-                return this.solutionFileContent?.split('\n').slice(entry.startLine! - 1, entry.startLine! + entry.lineCount! - 1);
-            })
-            .filter((line) => line && line.trim().length !== 0).length;
+    readonly removedLineCount = computed(
+        () =>
+            this.diffEntries()
+                .filter((entry) => entry && entry.previousFilePath && entry.previousStartLine && entry.previousLineCount)
+                .flatMap((entry) => {
+                    return this.originalFileContent()
+                        ?.split('\n')
+                        .slice(entry.previousStartLine! - 1, entry.previousStartLine! + entry.previousLineCount! - 1);
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)